tests/Tests/EventStoreTest.php

```diff
@@ -315,6 +315,27 @@ public function it_can_create_a_backward_iterator()
     }
 
     /**
+     * @test
+     */
+    public function it_can_process_the_all_stream_with_a_forward_iterator()
+    {
+        $client = new \GuzzleHttp\Client([
+            'auth' => ['admin', 'changeit'],
+            'handler' => new \GuzzleHttp\Handler\CurlMultiHandler(),
+        ]);
+        $httpClient = new GuzzleHttpClient($client);
+        $this->es = new EventStore('http://127.0.0.1:2113', $httpClient);
+
+        $this->prepareTestStream(1);
+        $streamName = rawurlencode('$all');
```

Comment:
You mean the `rawurlencode()`? Not sure how to provide a backward-compatible version here - if users currently encoded manually already, it'll be encoded twice then...